### PR TITLE
ePBS wiki page

### PR DESCRIPTION
**Pull request by thogiti**
I am writing a detailed wiki page for ePBS after reading and watching a ton references. My brain is a bit fried after taking so much notes. Now, I want to pull in the notes in some logical structure into one wiki page. Please guide me if this is a good direction for our wiki.

My goal is to have one page for ePBS that can contain everything and should be accessible to beginners and intermediate knowledgeable audience. It will be a one stop to learn everything about ePBS. 

https://github.com/thogiti/protocol-studies/blob/pbs/docs/wiki/research/PBS/ePBS.md 

P.S. I will be adding some visuals once I feel content is complete.

### CODE REVIEW

**Review by taxmeifyoucan — COMMENTED** [review]
Thanks for diving into ePBS! I left few comments, first part of my review, I will comment more later. Generally I need to suggest you to first check the current pbs doc by @arynyestos which often overlaps this.

File: docs/wiki/research/PBS/ePBS.md [hunk 1]
```diff
@@ -0,0 +1,399 @@
+# Envisioning Ethereum's Future: The Path to Enshrined Proposer-Builder Separation (ePBS)
+
+## [DRAFT MODE - This is work in progress.]
+
+## [ELI5](#eli5)
+
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md:6]: Ok, thx. Done.

File: docs/wiki/research/PBS/ePBS.md [hunk 2]
```diff
@@ -0,0 +1,399 @@
+# Envisioning Ethereum's Future: The Path to Enshrined Proposer-Builder Separation (ePBS)
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md:1]: Ok, thx. Done.

File: docs/wiki/research/PBS/ePBS.md [hunk 3]
```diff
@@ -0,0 +1,399 @@
...
+- **Efficiency and Specialization**: Separating the roles allows each actor to specialize in their respective tasks, leading to more efficient block construction and potentially higher network throughput.
+- **Fairness and MEV Distribution**: PBS can help in creating a more competitive and fair environment for MEV extraction, as multiple builders compete to have their blocks chosen by proposers.
+
+PBS is fundamentally a recognition that different roles in the blockchain ecosystem can have distinct capabilities and incentives, and that separating these roles can lead to improved efficiency, security, and fairness (division of labor). This concept has been explored and implemented in various ways, including through external services like MEV-Boost in Ethereum, which facilitates the interaction between proposers and builders outside the core protocol.
+
+## [Brief overview of ePBS](#brief-overview-of-epbs)
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md:99]: Ok, thx. Merged my changes to the `pbs.md` page.

File: docs/wiki/research/PBS/ePBS.md [hunk 4]
```diff
@@ -0,0 +1,399 @@
...
+- **Builders** are entities or algorithms that assemble blocks, optimizing the transaction order for profitability (e.g., maximizing MEV extraction) and offering these blocks to proposers through a transparent auction mechanism.
+- **Relays**, in the traditional MEV-Boost context, serve as intermediaries that facilitate the communication between proposers and builders. Under ePBS, the reliance on external relays is diminished or redefined, as the protocol itself facilitates the direct interaction between proposers and builders.
+
+The need to enshrine PBS into Ethereum's protocol stems from the desire to mitigate risks associated with external, trust-dependent services, improve the decentralization and security of block production, and better manage the complexities around MEV extraction. By formalizing these roles within the Ethereum protocol, ePBS seeks to ensure a fair, efficient, and transparent process for transaction ordering and block construction, aligned with Ethereum's ethos of decentralization and open participation.
+
+
+### [PBS in historical context](#pbs-in-historical-context)
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md:113]: Ok, thx. Done.

File: docs/wiki/research/PBS/ePBS.md [hunk 5]
```diff
@@ -0,0 +1,399 @@
...
+- [Notes on Proposer-Builder Separation PBS](https://barnabe.substack.com/p/pbs)
+- [Beyond the Basics: The Unanticipated Advantages of ePBS](https://hackmd.io/@potuz/ry9NirU2p)
+- [ePBS design constraints](https://ethresear.ch/t/epbs-design-constraints/18728/1)
+- [Payload Timeliness Committee](https://ethresear.ch/t/payload-timeliness-committee-ptc-an-epbs-design/16054)
+- [PEPC](https://ethresear.ch/t/unbundling-pbs-towards-protocol-enforced-proposer-commitments-pepc/13879)
+- [An Incomplete Guide to PBS - with Mike Neuder and Chris Hager](https://www.youtube.com/watch?v=mEbK9AX7X7o)
+- [ePBS Breakout Room](https://www.youtube.com/watch?v=63juNVzd1P4)
+- [mev-bibliography](https://github.com/michaelneuder/mev-bibliography#readme)
+
+**Table of Contents**
```
[the remark this answers is not in the document]
Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md]: Great, thx. Looks like we don't need a separate TOC.

**Comment by thogiti** [comment]
I've included a section to explain the Payload-Timeliness Committee (PTC) in ePBS page. However, given that specific implementation details for the PTC are now under discussion, it might be beneficial to dedicate a separate page for it to cover the information comprehensively.

File: docs/wiki/research/PBS/ePBS.md [hunk 6]
```diff
@@ -0,0 +1,398 @@
+# Enshrined Proposer-Builder Separation (ePBS)
+
+## [DRAFT MODE - This is work in progress.]
```
Comment (taxmeifyoucan) [inline docs/wiki/research/PBS/ePBS.md:3]: Is it still WIP?

Comment (thogiti) [reply docs/wiki/research/PBS/ePBS.md:3]: I want to write a brief section for PEPC similar to PTC and TBHL and remove WIP label. 

Is that ok?

Comment (taxmeifyoucan) [reply docs/wiki/research/PBS/ePBS.md:3]: Of course, I was just checking

**Comment by thogiti** [comment]
@taxmeifyoucan I completed the ePBS page. Can you please review and signal if it os ok to be merged.
